### PR TITLE
bugfix | missing contextual message

### PR DESCRIPTION
**Pull request by omarCreativeDev**
- added missing contextual success message
- tweaks to html css class's to get spacing perfect
- ensure Upload files and remove files cta's don't display if files are uploading or files have already been uploaded

### CODE REVIEW

File: src/app/features/bulk-upload/files-upload/files-upload.component.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
 
   <app-selected-files-list *ngIf="!filesUploaded"></app-selected-files-list>
 
-  <div *ngIf="!filesUploading">
+  <div class="govuk-!-margin-top-6" *ngIf="!filesUploading && !filesUploaded">
```
Comment (ChazUK) [inline src/app/features/bulk-upload/files-upload/files-upload.component.html]: Still not sure I agree with this hiding the Upload button part, disabling it makes sense. But understand it's a design thing

File: src/app/features/bulk-upload/selected-files-list/selected-files-list.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<table class="govuk-table" *ngIf="selectedFiles?.length">
+<table class="govuk-table govuk-!-margin-top-6" *ngIf="selectedFiles?.length">
```
Comment (ChazUK) [inline src/app/features/bulk-upload/selected-files-list/selected-files-list.component.html]: On all the other GOV.UK Elements the spacing between them is pretty much always handled by the margin on the bottom. Is it possible to keep that consistent?

**Comment by omarCreativeDev** [comment]
[When loaded files pass validation, correct details and notification not received](https://trello.com/c/y1K7XqrO/90-when-loaded-files-pass-validation-correct-details-and-notification-not-received)

**Comment by omarCreativeDev** [comment]
[(1) Bulk Upload - Passed Validation with No Warnings](https://trello.com/c/B7rG0kit/69-1-bulk-upload-passed-validation-with-no-warnings)

File: src/app/features/bulk-upload/files-upload/files-upload.component.html (hunk 1 above)
Comment (omarCreativeDev) [reply src/app/features/bulk-upload/files-upload/files-upload.component.html]: I spoke to Rich about this yesterday and he said we should be hiding it. I suppose we can always discuss tomorrow when all of us are int he office and we can try to sway him to leave the button and disable instead...?

File: src/app/features/bulk-upload/selected-files-list/selected-files-list.component.html (hunk 2 above)
Comment (omarCreativeDev) [reply src/app/features/bulk-upload/selected-files-list/selected-files-list.component.html]: good point, however i had to tweak the css classes for margin etc due to the inset-text component having both top and margin bottom. Please see `node_modules/govuk-frontend/components/inset-text/_inset-text.scss:28`

**Pull request merged**